### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by dcsobral**
Remove examples based on deprecated and removed scala.concurrent.jolib.

Drop extraneous semi-colons, plus System.out and Console. prefixes.

### CODE REVIEW

File: examples/src/main/scala/fors.scala [hunk 1]
```diff
@@ -27,9 +27,11 @@ object fors {
...
 
   def findNums(n: Int): Iterable[(Int, Int)] =
-    for (i <- 1 until n;
-         j <- 1 until (i-1);
-         if isPrime(i+j)) yield (i, j)
+    for {
+      i <- 1 until n
+      j <- 1 until (i-1)
+      if isPrime(i+j)
+    } yield (i, j)
```
Comment (jsuereth) [inline examples/src/main/scala/fors.scala:34]: do you prefer this over `i -> j`?

File: examples/src/main/scala/maps.scala [hunk 2]
```diff
@@ -171,7 +171,7 @@ object maps {
 
     override def equals(that: Any): Boolean =
```
Comment (jsuereth) [inline examples/src/main/scala/maps.scala:172]: pattern match here instead?

File: examples/src/main/scala/monads/simpleInterpreter.scala [hunk 3]
```diff
@@ -53,26 +53,27 @@ object simpleInterpreter {
   def interp(t: Term, e: Environment): M[Value] = t match {
     case Var(x) => lookup(x, e)
     case Con(n) => unitM(Num(n))
-    case Add(l, r) => for (a <- interp(l, e);
-			   b <- interp(r, e);
-			   c <- add(a, b))
-                      yield c
+    case Add(l, r) => for {
+                        a <- interp(l, e)
```
Comment (jsuereth) [inline examples/src/main/scala/monads/simpleInterpreter.scala:57]: tab vs. space issue here, it seems.

File: examples/src/main/scala/monads/simpleInterpreter.scala [hunk 4]
```diff
@@ -53,26 +53,27 @@ object simpleInterpreter {
...
+			b <- interp(r, e)
+			c <- add(a, b)
+                      } yield c
     case Lam(x, t) => unitM(Fun(a => interp(t, Pair(x, a) :: e)))
-    case App(f, t) => for (a <- interp(f, e);
-			   b <- interp(t, e);
-			   c <- apply(a, b))
-		      yield c
+    case App(f, t) => for {
+                        a <- interp(f, e)
```
Comment (jsuereth) [inline examples/src/main/scala/monads/simpleInterpreter.scala:63]: ditto above.

File: examples/src/main/scala/monads/stateInterpreter.scala [hunk 5]
```diff
@@ -62,26 +63,28 @@ object stateInterpreter {
   def interp(t: Term, e: Environment): M[Value] = t match {
     case Var(x) => lookup(x, e)
     case Con(n) => unitM(Num(n))
-    case Add(l, r) => for (a <- interp(l, e);
-			   b <- interp(r, e);
-			   c <- add(a, b))
-                      yield c
+    case Add(l, r) => for {
+                        a <- interp(l, e)
```
Comment (jsuereth) [inline examples/src/main/scala/monads/stateInterpreter.scala:67]: again with the spacing...

**Comment by jsuereth** [comment]
Everything looks great!   THanks much.   I added a few notes, mostly for myself, of potential improvements. If you have a chance to get to them, great.  Otherwise I'll pull this sometime before M4.

File: examples/src/main/scala/fors.scala (hunk 1 above)
Comment (dcsobral) [inline examples/src/main/scala/fors.scala:34]: I didn't look at that. I was just removing semi-colons. Generally, I only like `i -> j` if it's a key/value relationship.

File: examples/src/main/scala/maps.scala (hunk 2 above)
Comment (dcsobral) [inline examples/src/main/scala/maps.scala:172]: I don't think so. A pattern matcher where all patterns are `_`  with a guard are not my thing.

File: examples/src/main/scala/monads/simpleInterpreter.scala (hunk 3 above)
Comment (dcsobral) [inline examples/src/main/scala/monads/simpleInterpreter.scala:57]: I knew something was fishy, but I didn't realize I left it broken. Do you want me to add an edit to this pull, or fix in another pull?

File: examples/src/main/scala/maps.scala (hunk 2 above)
Comment (jsuereth) [inline examples/src/main/scala/maps.scala:172]:
``` scala
override def equals(that: Any): Boolean = that match {
   case o: Date => day == o.day && month = o.month && year == o.year
   case _ => false
}
```

I find that way more clear than what's there.

Comment (dcsobral) [inline examples/src/main/scala/maps.scala:172]: Oh, of course! I was looking above, not below.

**Pull request merged**